### PR TITLE
[dualtor-mixed] Set grpc config as insecure for testing

### PR DESCRIPTION
**Pull request by lolyu**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [x] 202205

#### What is the motivation for this PR?
Use `insecure` config for grpc connections from `ycabled` in tests.

#### How did you do it?
Modify `/etc/sonic/grpc_secrets.json`

#### How did you verify/test it?
This depends on https://github.com/sonic-net/sonic-platform-daemons/pull/298

### CODE REVIEW

File: ansible/config_sonic_basedon_testbed.yml [hunk 1]
```diff
@@ -525,6 +525,42 @@
...
+
+          - name: save new grpc setting
+            copy:
+              content: "{{ grpc_secrets | to_nice_json }}"
+              dest: "{{ grpc_credential_file }}"
+
+          when: stat_result.stat.exists
+
+        when: "'dualtor-mixed' in topo"
+
```
Comment (vdahiya12) [inline ansible/config_sonic_basedon_testbed.yml:563]: hi @lolyu can you explain how docker restart pmon is executed in this case ?

Comment (lolyu) [reply ansible/config_sonic_basedon_testbed.yml:570]: Added, thanks!

**Comment by yxieca** [comment]
@lolyu  why did you set the included in 202205 branch flag? This PR hasn't been merged yet.

**Comment by yxieca** [comment]
@lolyu  did you test it on a device with and/or with the file to change? Are behaviors in both scenarios good?

**Review by vdahiya12 — APPROVED** [review]

**Comment by lolyu** [comment]
> @lolyu did you test it on a device with and/or with the file to change? Are behaviors in both scenarios good?

Tested, and fix a issue of file parsing.

**Review by bingwang-ms — APPROVED** [review]

**Pull request merged**